File: tests/openmp_examples/devices/teams.1.c
#pragma omp begin declare variant match(device={kind(host)})
#pragma omp end declare variant
#pragma omp begin declare variant match(device={kind(nohost)})
#pragma omp end declare variant
#pragma omp target map(to: B[:N], C[:N]) map(tofrom: sum0, sum1)
#pragma omp teams num_teams(2)
#pragma omp parallel for reduction(+:sum0)
#pragma omp parallel for reduction(+:sum1)

File: tests/openmp_examples/devices/teams.1.f90
!$omp     target map(to: B, C) map(tofrom: sum0, sum1)
!$omp     teams num_teams(2)
!$omp          parallel do reduction(+:sum0)
!$omp          parallel do reduction(+:sum1)
!$omp     end teams
!$omp     end target
